iphone/Classes/TiUIListView.m

```diff
@@ -1969,6 +1971,14 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView
           [eventArgs setValue:NUMINTEGER([indexPath section]) forKey:@"firstVisibleSectionIndex"];
           [eventArgs setValue:section forKey:@"firstVisibleSection"];
           [eventArgs setValue:[section itemAtIndex:[indexPath row]] forKey:@"firstVisibleItem"];
+
+          if (lastVisibleItem == [indexPath row] && lastVisibleSection == [indexPath section]) {
+            // skip event - still the same item at the top
+          } else {
+            [self.proxy fireEvent:@"scrolling" withObject:eventArgs propagate:NO];
```

Comment:
Simplify the if-statement to `lastVisibleItem != [indexPath row] || lastVisibleSection != [indexPath section]`?